### ISSUE TITLE
False positive on "Uses passive event listeners"

### ISSUE THREAD

**Issue by pavelfeldman** [GoogleChrome/lighthouse#1614]
I was testing it on meduza.io

report:

![modern_js_features](https://cloud.githubusercontent.com/assets/883973/22571449/a1245ec4-e954-11e6-874f-9d2ecf2c534c.png)

listeners:

![blocking_listeners](https://cloud.githubusercontent.com/assets/883973/22571527/eb14f5a2-e954-11e6-9ca7-479ec0bf6440.png)

**Comment by ebidel** [comment]
AFAIK, passive event listeners are for touch and wheel events. We maintain a whitelist [here](https://github.com/GoogleChrome/lighthouse/blob/a10f6e0a31819704634014aed831165da43bcf41/lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js#L34), which doesn't include the `scroll` event. Should we be looking at that as well?

**Comment by pavelfeldman** [comment]
No, you are right, scroll is async, passive does not apply there. Let me reformulate the report:

I have no passive listeners for wheel or touch, and lighthouse is saying that I use passive listeners for those.

**Comment by paulirish** [comment]
I think the wording that would make more sense to pavel is..

> All touch/wheel listeners are marked as passive

Because the page itself isn't using any passive listeners, so a checkmark on "uses" is confutso.

**Comment by ebidel** [comment]
Gotcha. 

Maybe this is a bigger question on how we present passing audits to the user? Right now, we show a "pass" when nothing in the audit is violated. IOW, the user's page may never use `document.write`. The fact that they didn't use it means they pass.

The "Avoids using X" cases don't suffer from the same subtly. You're avoiding X if you're not using it :)
